ext/standard/basic_functions.stub.php

```diff
@@ -60,6 +60,24 @@ function stream_wrapper_restore(string $protocol): bool {}
 /** @return int|false */
 function array_push(array &$stack, ...$args) {}
 
+function krsort(array &$stack, int $sort_flags = PHP_SORT_REGULAR): bool {}
+
+function ksort(array &$stack, int $sort_flags = PHP_SORT_REGULAR): bool {}
+
+function count(mixed $array, int $mode = COUNT_NORAML): bool {}
```

Comment:
``mixed`` is not a valid type hint, moreover doesn't this need a doc comment ``@param array|Countable $array`` ?

Reply:
sorry, I've add doc comment @param array|Countable.